### PR TITLE
Update package.json and CHANGELOG for release 1.4.0

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Review by Blackbaud-AlexKingman — DISMISSED** [review]

**Review by blackbaud-johnly — DISMISSED** [review]
LGTM

**Review by blackbaud-johnly — APPROVED** [review]
Re-approving after updating the date since this wasn't merged Friday.

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**
